drivers/display/display_mcux_elcdif.c

```diff
@@ -203,8 +203,10 @@ static int mcux_elcdif_write(const struct device *dev, const uint16_t x, const u
 	/* Update index of active framebuffer */
 	dev_data->next_idx = (dev_data->next_idx + 1) % CONFIG_MCUX_ELCDIF_FB_NUM;
 #endif
-	/* Enable frame buffer completion interrupt */
+
+#ifdef CONFIG_MCUX_ELCDIF_LP
```

Comment:
use `if(IS_ENABLED(...))`

Reply:
same for other occurrences

Reply:
Thanks for your suggestions, I've changed the macros with your suggestions.

Reply:
I meant to use C `if`, and avoid preprocessor #if